Dockerfile

```
@@ -2,18 +2,15 @@ FROM ubuntu:18.04
 MAINTAINER Lisk <sysops@lightcurve.io>
 ENV DEBIAN_FRONTEND noninteractive
 
-ARG user_name
 ARG user_id
 
-RUN useradd --create-home --uid $user_id $user_name
+RUN useradd --create-home --uid $user_id liskjs
 RUN apt-get update && apt-get install --assume-yes libtool automake autoconf curl python-minimal build-essential git vim || true
```

@michielmulders  this `|| true` might cause problems; why no fail fast?